### PR TITLE
Set up a Hello Microverse project

### PR DESCRIPTION
**Pull request by saadat-ali-awan**
Completed Task For First Review

### CODE REVIEW

**Review by aliabbani — CHANGES_REQUESTED** [review]
Hi @Saadat123456 ✋, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! 

To Highlight! 🟢
You created a new GitHub repo. ✔️
You created a new branch according to the GitHub flow rules. ✔️
You added a .gitignore file. ✔️
Opened your first pull request. ✔️

## Required Changes ♻️ 

Check the comments under the review. 
- [x] Please add a test.md file and please write a simple text inside it "This file should be ignored by git".
- [x] Please Set up a linter correctly. Please check [this link ](https://github.com/microverseinc/linters-config/tree/master/html-css) for more information. Also, please make sure that the Linter result is green on the pull request. Linter should always report no errors when you submit a future pull request (on this pull request the Stylelint is showing ❌).
- [x] Please make sure to write meaningful commit messages in your next commit. You can check [this link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more details.
- [x] Pull Request should have a descriptive title telling what changes are inside. You can change the Title from (First Review) to (Set up a Hello Microverse project)


## Optional suggestions 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by saadat-ali-awan** [comment]
Hello @aliabbani !
I carefully went through your suggestions and would be careful next time.
Regards Saadat

**Comment by saadat-ali-awan** [comment]
Hello @aliabbani !
I went through all the changes and have fixed all the changes.

1. The test.md file was added in the previously and its name was mentioned in the .gitignore file. That's why it is not uploaded on the GitHub.
2. I have setup linter correctly and it is working fine.
3. The commits messages have been improved on new commits.
4. Pull request has been renamed to "Set up a Hello Microverse project".
Regards Saadat

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (mmsesay) [inline test.md]: - [ ] Just as the content of the file reads, you should ignore this file. I noticed you ignored it in your `.gitignore` file but I believe it must have been cached before your stage your changes.

Please clear the cached `git rm -r --cached .` 
Stage your files and push them again to github.
Kindly verify that the file is not present in your repo before asking for another review.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Saadat123456 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md (hunk 1 above)
Comment (saadat-ali-awan) [reply test.md]: Thanks @mmsesay!
I have followed what you said and the file test.md has been ignored.

**Comment by elmar8287** [comment]
Hi @Saadat123456, 

Well done on making the suggested changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

My suggestions: 

- Please, add node_modules into .gitignore file
- Make your README file more professional, considering the Microverse template
- Fix the live demo link - it is not correctly set. You can set it from SETTINGS => Pages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by saadat-ali-awan** [comment]
Hello @elmar8287!
I have taken the notes and will do so. Thanks for suggestions!
Regards Saadat

**Pull request merged**